### DISCUSSION TITLE
奇忽局宣言 1.0 (持续更新迭代）

### DISCUSSION THREAD

**Discussion by midai17** [qihuju/qihuju.github.io#33] [Ideas]
@Tz-H 天资为奇忽局起草了**奇忽局宣言1.0**：

我们是比特荒野上的殖民者，也是指路人；
我们是赛博朋克原住民的血族，也是他们的导师；
我们回溯往矣，拥抱当下，探索未来；
我们是奇忽局。

*We are the colonists and the pathfinders in the Byte Wilderness.
We are the bloodline and mentors of the Cyberpunk Natives. 
We are looking after the past, embracing the present, and exploring the future.
We are the Singularity Fantasy Bureau.*

**（持续更新迭代 working-in-progress...）**

**Comment by Lupingyue** [comment]
抢到沙发🛋️✌️

为什么 @midai17 把“奇忽局”写成“奇乎局”，好像有一次还写成“奇勿局”，是故意的谐音梗吗😄😄

**Reply by Tz-H** [reply]
估计是笔误。。。已修订

**Reply by Lupingyue** [reply]
说不定是故意的呢😄，如果是故意的也蛮有趣味的，听听 @midai17 来说说😄

**Reply by midai17** [reply]
我们就是我们。我们想成为什么，我们就是什么。我们的肉体早已被侵略，我们的灵魂仍然在挣扎，我们用慧脑扫描彼此的频率与序列号，我们都属于奇hu局。

哈哈，typo一定是出于我们成员地域和活动渠道多样性，也集“奇忽/乎/勿“呼局山寨文化之大成。😬

**Reply by Lupingyue** [reply]
> 我们就是我们。我们想成为什么，我们就是什么。我们的肉体早已被侵略，我们的灵魂仍然在挣扎，我们用慧脑扫描彼此的频率与序列号，我们都属于奇hu局。
> 
> 哈哈，typo一定是出于我们成员地域和活动渠道多样性，也集“奇忽/乎/勿“呼局山寨文化之大成。😬

霸气👍

**Comment by Tz-H** [comment]
感谢并期待奇忽局宣言能在1.0版本的基础上激荡出更绚烂的烟花~

**Reply by Sleeper2060** [reply]
👀有幸邀请你参加宣言共同协作版的第一块区块吗？
请移步[这里](https://github.com/qihuju/qihuju.github.io/projects/4)进行了解。

**Reply by Lupingyue** [reply]
> 👀有幸邀请你参加宣言共同协作版的第一块区块吗？
> 请移步[这里](https://github.com/qihuju/qihuju.github.io/projects/4)进行了解。

这个计划很酷，估计 @MingxuanXie @Tz-H @askender 都会很有兴趣

**Reply by Tz-H** [reply]
提供一个工具：

[The String Converter - Hash, Encode And Decode Strings Using Any Known Technique](https://www.codepunker.com/tools/string-converter)

它可以把给定的一段文字转化为它所对应的哈希值（Hashcode）

例如：

> 我们是比特荒野上的殖民者，也是指路人；
> 我们是赛博朋克原住民的血族，也是他们的导师；
> 我们回溯往矣，拥抱当下，探索未来；
> 我们是奇忽局。
> 
> We are the colonists and the pathfinders in the Byte Wilderness.
> We are the bloodline and mentors of the Cyberpunk Natives.
> We are looking after the past, embracing the present, and exploring the future.
> We are the Singularity Fantasy Bureau.

它的哈希值就是：

```
 "md5": "15fa98ae55083a1e8993a14d284c1f75",
 "sha1": "d6d749091065888048f40007a590698f0bd50c09",
 "sha256": "6430b6f39a9ed1800febd232f898a425327dc731bbd85440750f68af73e77fbb",
 "sha384": "cf0bf8560579957265b30a0e0fc5f420f8673b8841b8645f8d435ab7f90c680eb4b295e6a991bcad8c1072ef248ee7f6",
 "sha512": "26ce11528a0ecf452696862e1d759f7c11ecf22e6a290e90ba0c2a8fbda1dfba163d894694cfdd9cfe0625dfa26293832e6ad5241d525c558341a5eb28aad66e",
 "md2": "8f89b68aee0c6b9d94d00b46e537c364",
 "md4": "bfb316bafc477978016cc0d3467d7430",
 "ripemd128": "e597b0c70711dc40c6c7fb29afea76bc",
 "ripemd160": "e233542af1a927ada1e040b537cbab3794c0adf1",
 "ripemd256": "2d573f6c629d83dbadb908c437787697ba82f6d7b09f5dcacb690cb62b7c441b",
 "ripemd320": "5e37bbe9cf3d53ae7c8049bf7210e20bc2360b63115e18d6275927ebf9d277db07c9877a2ead1643",
 "whirlpool": "409b1034bdd5a72fcb30fa4f641ee6b79bdea77cb514c40460d004f3f38a714bc19550c495f19890ce9d3084bf58609ecfaf7fd655a2a62e5c1f7aba89dc1433",
 "tiger128,3": "5d22d49daf369f5f413b7c54197bc201",
 "tiger160,3": "5d22d49daf369f5f413b7c54197bc201a413e9c7",
 "tiger192,3": "5d22d49daf369f5f413b7c54197bc201a413e9c728fc560f",
 "tiger128,4": "b3618e9000ef8abb4b291079527d312b",
 "tiger160,4": "b3618e9000ef8abb4b291079527d312b82c0d0f0",
 "tiger192,4": "b3618e9000ef8abb4b291079527d312b82c0d0f09362d6f5",
 "snefru": "5e0b135bf4f0511e660f89f9bce37f7468b3372ebd65bd9fbf7c0af43e2cfa4d",
 "gost": "43490b15de692152a83d26c4f1464e2119e746864f4ca4e167dfb0fb9d940a9b",
 "adler32": "b42ce3c5",
 "crc32": "290de833",
 "crc32b": "90d56297",
 "haval128,3": "1c32caee39e571b94575bee404dff9c0",
 "haval160,3": "cf889fa96b1107a849a1dddb7f8c49ba2098f0cc",
 "haval192,3": "3ab29a9e8c75a73433ac81952b374033281545223e174cf0",
 "haval224,3": "814b0d46ba61280f2f024db47b10539631b9ecd07a7b3755e2c104be",
 "haval256,3": "f52312f94b620c0898f7962ea10bdde4b2b1e9e7f6bc22b1a4599c11cb9f1bee",
 "haval128,4": "9f8a79b67b2414a1565454a0963bcb26",
 "haval160,4": "1527203d7d55e9b7c98eb3fcdc26e7943eb6ed8d",
 "haval192,4": "b97338ad2bdc6c243f65c57600b0d02daa2d6761582b8de3",
 "haval224,4": "188da5e84d15e458e8dce248eec415fb4a54b35c057c221a273ada94",
 "haval256,4": "2df117d53d491d52c67369149638a92ae098c9dae519a8066509712a6cc64bbb",
 "haval128,5": "3ffc71204495b025623d0af8cbcc4ab2",
 "haval160,5": "8dc2dfb2de29af56bc3f98a2b47488863777ce81",
 "haval192,5": "390be7906a1c3d86c7968f3ffa276f92f3956dd03acd160c",
 "haval224,5": "91d94dad4bfd21e42fba46ddba055eaac648611eeda27b484394c39d",
 "haval256,5": "7a456905698ccb4f214fb2f7f065f8129fd6e93c950c1db3b925fb00d7aac154"
```

**Reply by Sleeper2060** [reply]
酷，马上修改一下。

**Comment by ppicassos** [comment]
小问题。 

我们回溯往矣，拥抱当下，探索未来
We are looking after the past, embracing the present, and exploring the future.

我理解“回溯”有几个意思：reminisce, recollect, look back, backtrack, retrace, , relive.
好像 look after 和原意有差距。

**Reply by Tz-H** [reply]
> 小问题。
> 
> 我们回溯往矣，拥抱当下，探索未来
> We are looking after the past, embracing the present, and exploring the future.
> 
> 我理解“回溯”有几个意思：reminisce, recollect, look back, backtrack, retrace, , relive.
> 好像 look after 和原意有差距。

感谢！

@Sleeper2060 @Lupingyue 

问题：如果我们想把宣言1.0升级到1.0.1，包括这样一个英文翻译的改进，在[宣言共同协作版](https://github.com/qihuju/qihuju.github.io/projects/4)处应如何进行区块链模拟操作？

**Reply by Lupingyue** [reply]
感觉只有 @askender 能回答这个问题

**Reply by ppicassos** [reply]
岳老师， “回溯以往”和 “回溯往矣” 哪个更好？

**Reply by Lupingyue** [reply]
> 岳老师， “回溯以往”和 “回溯往矣” 哪个更好？

ppicassos是群里的哪位？你们的avatar怎么都不统一😒

**Reply by Lupingyue** [reply]
> 岳老师， “回溯以往”和 “回溯往矣” 哪个更好？

往矣似乎更优雅一点

**Reply by askender** [reply]
「往矣」是动词加助词。比如在「俱往矣」「虽万千人吾往矣」等，这里当做名词感觉略别扭，含义是「往昔、过往」，但别扭也是某种种错位的艺术吧。

另外，「区块链模拟」随意玩for fun即可，通常链是不修改的，为了强行解释：1 分叉了  2 被51%攻击了 3 源码漏洞 4 还在测试网  5 补上创世区块作为第0区块，声明只有hash不可更改，文字内容可更新的链    6 可以pull request和merge的dag图作为链

p.s.
宣言有趣有力量就好，我瞎评论一下别介意：
比特荒野 - 也许是比特废墟，毕竟已经有繁华过的web2了   殖民者-意味着统治原住民?实际是拓荒者。
指路人、导师 - 为谁指路，谁需要指路，似乎有一丝后浪的感觉
赛博朋克原住民的血族 - 似乎没有新的含义， 血族似乎有些生硬,  赛博朋克也有高科技和低生活的意味，只是一种可能的近未来
我们回溯往矣，拥抱当下，探索未来 - 这句话略微有点空

本来宣言就有某种口号的气质，或者是一种让人有印象的意象，从这一点上，奇忽局的宣言做到了。
无产者在这个革命中失去的只是锁链。他们获得的将是整个世界。全世界無產者，聯合起來！
我要这天再遮不住我眼，要这地再埋不了我心。要这众生都明白我意，要那诸佛都烟消云散
四月是最残忍的季节，荒地上长着丁香，把回忆和欲望掺杂在一起。 艾略特《荒原》 April is the cruelest month 

可能我的想法比较沉重一些，我偏好类似创刊词、白皮书、论文、或者简洁的一句话或者诗的写法。 
一句话的写法，要么务虚和文艺，要么直白。我的理解是，宣言表达了这里是下一代赛博空间的探路者的聚集地。

如何定义「探索未来」？  后续的扩展包中， 如果能描述困惑和困境，提出几种可能的解决方案，这样整体才会更充实一些。 

对于我来说，技术的华丽和换代只是表象，内在的是科学的突破、思维方式和协作方式的进化、人类对于趣味、艺术和终极问题探索的过程。

**Comment by Tz-H** [comment]
阿里达摩院基于GPT-3的中文语言训练模型刚刚放出了，在这里可以进行线上测试：

[https://nlp.aliyun.com/portal#/BigText_chinese](https://nlp.aliyun.com/portal#/BigText_chinese)

当输入文本是奇忽局宣言1.0：

> 我们是比特荒野上的殖民者，也是指路人；
> 我们是赛博朋克原住民的血族，也是他们的导师；
> 我们回溯往矣，拥抱当下，探索未来；
> 我们是奇忽局。

由该 AI 引擎所续写的故事片段如下：

AI 续写其一：

> 比特荒野中有一个小城市，这座城市以一种特别荒芜的方式存在着。这里没有人，也没有任何居民。然而，在这个城市的某一个角落里，却有一个奇怪的生物，正在吞噬着任何人类的生命。他们没有血缘，只是一种比较亲近的关系，他们的能力是用人类的生命做成。他们不属于人类文明，不是原住民文明，他们是一种全新的文明，他们的文明是从未有人的时代产生的，这个文明的文明来自于原始的文明。他们的文明是从最古老的人类文明产生的，这个文明的产物，就是一种人类的血液。在人类文明产生以前，这个文明的文明就是人类文明的一部分，他们的文明发展

AI 续写其二：

> 这是一场没有硝烟的战争，这是一场人类的文明之争；是一场人类的信仰与追求的战争；是一场不同文明的较量；是一场人类与信仰与文明的战争。在浩瀚的宇宙中，一个人、一个文明、一个国家，他们互相争斗又相互依存。这是一个没有硝烟的战争，这是一场不同文明的较量。你的生命，从你出生的那一刻起就注定了你不一样的命运。你的一生就是一个人的一生，一个民族的一生。不同的生命对你的影响力是不同的，不同的生命给你的影响也是不同的，你生来的命运，可能就决定你这一生的命运。“你们的命运，是不是决定于你们的出生？你们的命运，是不是决定于你们出生时的那个奇怪的“一“这个字？“我们的命运，是不是决定于人出生时的那个奇怪的“一“这两个字？那个奇怪的“一“字？就是因为它的一字之差而改变的，“命“也是命运。“在宇宙的浩瀚中，我们的命运都是不相同的，但是，我们的命运却有着无法预测的因缘。这就是因为我们的命运与别人的命运是无法用一个字来概括的。有人说：“你的出生是一场战争，你的命运是一个奇迹。“这句话是否有道理？没有道理。但是，我们出生的那一刻就注定了我们一生的命运。这句话的意思是说：不同的生命对你的影响力是不同的，不同的生命给你的影

AI 续写其三：

> 我们是赛博朋克，他们是我们的兄弟。我们是赛博朋克，他们是我们的朋友。我们可以在一起，我们一起打造全新的赛博朋克，创新全新的赛博朋克文化。赛博朋克，是一种自由思想的流淌，也是一种精神的传承。赛博朋克，是一种创新的精神，也是一种价值的追求。赛博朋克，是一种

参考资料：

 1. [知乎：达摩院用128张GPU烧出“中文版GPT-3”，我试了下，这文风不是开往幼儿园的车…
](https://zhuanlan.zhihu.com/p/365999690)

2. [阿里云：如果让AI来续写这些小说……](https://mp.weixin.qq.com/s/LiuSwrGwHwfEIHjGSR3LKw)

**Reply by Lupingyue** [reply]
人机协作的人机未来宣言

**Comment by MingxuanXie** [comment]
添加一个我最近做的一个艺术项目的宣言  [画廊必须燃烧](https://mingxuan.fun/work/2021/04/17/gallery-must-blaze.html)
看看能不能结合一下

---
你能透過無數白盒子空間來判斷當今畫廊有多麽無聊。
那些被看門人管控下的美術館在後現代的虛假表皮下搖搖欲墜。

我無意於建造門檻，抑或表皮。
我厭倦了現實空間的條條框框（尤其是消防規範）。
因為對於藝術，我只想保留最令人不安的部分。

我想要畫廊淩駕於於物理限制。
畫廊要虛擬，開源，去中心，不可替換。
要在資本泡沫中乘風破浪，
又在元宇宙中四分五裂。
要在公眾號、INS與P站之間來回跳轉，
又在區塊鏈上凝聚共識。

_“下一個巴比倫”將是建立在Babylon.js之上的“新巴比倫”？_

畫廊必須穿越次元————
嗨、萌、好玩、耍梗、互動、沈迷、性轉、爆炸、全真、被駭、故障、分布自治、病毒性傳播，
無論線上或線下。

要冷，便寒如北極冰川代碼庫。
要熱，就燙過極限超頻的芯片。

畫廊必須燃燒🔥

You can judge how boring today’s galleries are by looking at these white boxes.
The gatekeeper-controlled galleries struggled behind the post-modern façades.

I have no desire to build gates or façades.
I am tired of the limitations in the real world (especially fire regulations).
Because with art, I only want to keep the most disquieting part.

I want galleries beyond physical constraints.
Galleries that virtualised, open-sourced, decentralised, and noun-fungible.
Galleries that surfing upon capital bubbles,
and dissolved in the metaverse.
Galleries that redirected among WeChat, Instagram and PornHub…
while building consensus on blockchains.

_Would “The Next Babylon”[1] be the “New Babylon”[2] built on Babylon.js?_

Galleries have to be trans-dimensional,
hyped, moe, playful, memeful, interactive, addictive, explosive, transsexual, hyperreal, hacked, glitched, DAO and going viral. Online or offline.

If cold, then cold as the Arctic Code Vault.
If hot, then hot as the overclocked chips.

Gallery must blaze🔥

---
该项目/宣言参考自Coop Himmelb(l)au的Architecture Must Blaze (1980).
1. Nieuwenhuys, Constant (1959-74). New Babylon.
2. Novak, Marcos (1998). “Next Babylon: Accidents to Play in”. The Art of the Accident. Netherlands: NAI Publishers/V2O̲rganisatie.

**Reply by Lupingyue** [reply]
画廊必须燃烧🔥，很燃🔥
